LazyProxy/LazyProxyBase.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+
+namespace LazyProxy
+{
+    /// <summary>
+    /// Base class for lazy proxies.
+    /// </summary>
+    public abstract class LazyProxyBase
+    {
+        /// <summary>
+        /// Initialize inner service of <see cref="Lazy{T}"/> by value factory.
```

Comment:
Initializes inner `<see cref="Lazy{T}"/>` instance with the `valueFactory` provided.